src/dataset-builder/CohortEditor.test.ts

```diff
@@ -436,25 +444,16 @@ describe('CohortEditor', () => {
     expect(screen.getByText('Group 1')).toBeTruthy();
   });
 
-  it.skip('can add/delete criteria groups and maintain user understandable names', async () => {
+  it('can delete criteria groups and maintain user understandable names', async () => {
     // Arrange
     showCohortEditor();
     const user = userEvent.setup();
-    // Assert
-    expect(screen.getByText('Group 1')).toBeTruthy();
     // Act
     await user.click(screen.getByText('Add group'));
-    // Assert
-    expect(screen.getByText('Group 2')).toBeTruthy();
-    // Act
-    await user.click(screen.getByLabelText('delete group 2'));
     await user.click(screen.getByText('Add group'));
     // Assert
     expect(screen.getByText('Group 1')).toBeTruthy();
     expect(screen.getByText('Group 2')).toBeTruthy();
-    // Act
-    await user.click(screen.getByText('Add group'));
-    // Assert
     expect(screen.getByText('Group 3')).toBeTruthy();
     // Act
     await user.click(screen.getByLabelText('delete group 2'));
```

Comment:
nit: in this test, is it worth adding some `toBeFalsy()` assertions? If the page loads with "Group 1", "Group 2", and "Group 3" text already present on the page, the test will pass regardless of which clicks it emulates.

I realize this is out of scope for this ticket, take this feedback as optional

Reply:
good idea!

Reply:
> is it worth adding some toBeFalsy() assertions? If the page loads with "Group 1", "Group 2", and "Group 3" text already present on the page, the test will pass regardless of which clicks it emulates

That's true, and there's no harm in doing this, but I don't think we do this in other cases. Personally I would test for cases we know can (or have) happened. If this problem ever comes up, and we need to fix it, we could add a test for it then.